### ISSUE TITLE
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.

### ISSUE THREAD

**Issue by emami7495** [cafebazaar/react-native-poolakey#5]
Hi, i got this error :

> The following dependencies do not satisfy the required version:
project ':cafebazaar_react-native-poolakey' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20

please update Kotlin Gradle plugin version

**Comment by iMohammad97** [comment]
no update on this?

**Comment by shiraliali** [comment]
It's very funny since Nov 22 the package is still not compatible :)))

**Comment by Jackpate-2003** [comment]
Unfortunately, the Kotlin version of this package has not been upgraded yet, but there is a solution to fix this error, changing the Kotlin version of our own project. If you are using Expo, first of all, if the android folder is created, delete it, then add this section to the **app.json** file:
```
"plugins": [
      [
        "expo-build-properties",
        {
          "android": {
            "kotlinVersion": "1.7.10"
          }
        }
      ]
]
```
Set the kotlinVersion property to version 1.7.10 or the latest version of Kotlin (currently stable version 1.8.22), now build the app again.

**Comment by sepehrsohrabii** [comment]
Apparently, the GitHub repo is up to date but the npm package is not. So you should install the package directly from GitHub repository. Using the bellow command solved the problem for me.
`npm install git+https://github.com/cafebazaar/react-native-poolakey.git`

**Comment by ahmousavi** [comment]
This problem fixed in version 3.1.0 of the package. If you are using an older version of React Native and have any problem in using version 3 you can use version 2.2.0 of the package as it is compatible with older React Native versions.
